src/main/java/com/iridium/iridiumskyblock/listeners/PlayerPortalListener.java

```diff
@@ -2,42 +2,42 @@
 
 import com.iridium.iridiumcore.utils.StringUtils;
 import com.iridium.iridiumskyblock.IridiumSkyblock;
+import com.iridium.iridiumskyblock.configs.Configuration;
+import com.iridium.iridiumskyblock.database.Island;
 import com.iridium.iridiumskyblock.managers.IslandManager;
 import com.iridium.iridiumskyblock.utils.LocationUtils;
 import org.bukkit.World;
 import org.bukkit.event.EventHandler;
 import org.bukkit.event.Listener;
 import org.bukkit.event.player.PlayerPortalEvent;
-import org.bukkit.event.player.PlayerTeleportEvent;
+import org.bukkit.event.player.PlayerTeleportEvent.TeleportCause;
 
 import java.util.Objects;
+import java.util.Optional;
 
 public class PlayerPortalListener implements Listener {
 
     @EventHandler
     public void onPlayerPortal(PlayerPortalEvent event) {
         IslandManager islandManager = IridiumSkyblock.getInstance().getIslandManager();
-        IridiumSkyblock.getInstance().getIslandManager().getIslandViaLocation(event.getFrom()).ifPresent(island -> {
```

Comment:
we need this, shouldn't send player while netherIslands enabled (it does on 2.6.7)

Reply:
no, if they arnt on the island it should just do them as normal

Reply:
I mean like our border issue
When player built portal on boundaries it was teleporting to normal nether, this will ignore that

Reply:
its will ignore sending player to normal nether/end when any issue

Reply:
I don't get your point, we don't have boundary issues anymore

Reply:
mabey, will have any new issues?

Reply:
You're confusing me... I get your point but I don't know why we should expect these problems to re-appear

Reply:
isn't possilbe? im just want blocking when couldn't find the island, may players can except the out of island and build portal on there?

Reply:
Also not possible. I'd just revert that too